fdbcli/BulkDumpCommand.actor.cpp

```diff
@@ -148,10 +167,10 @@ ACTOR Future<UID> bulkDumpCommandActor(Reference<IClusterConnectionRecord> clust
 
 CommandFactory bulkDumpFactory(
     "bulkdump",
-    CommandHelp("bulkdump [mode|local|clear|status] [ARGs]",
+    CommandHelp("bulkdump [mode|local|blobstore|clear|status] [ARGs]",
```

Comment:
In follow-on, can we change this so command is 'bulkdump mode|create|clear|status [ARGs]'?

Reply:
Ditto.